client/src/components/common/applicant-info-1.vue

```diff
@@ -603,7 +603,7 @@ export default class ApplicantInfo1 extends NameRequestMixin {
   nextAction () {
     this.validate()
     if (this.isValid) {
-      this.next()
+      newReqModule.mutateSubmissionTabNumber(this.submissionTabNumber + 1)
```

Comment:
This was in `next()` for this page only.

Reply:
This wonky architecture requires a developer to update things in MORE places rather than FEWER places when something needs to be changed :(

Reply:
Good change then!